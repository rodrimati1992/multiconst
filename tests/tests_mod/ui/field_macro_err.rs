use multiconst::field;

fn main(){
    let _: field!("hello");
    let _: field!(0x100);
    let _: field!(-1);
    let _: field!(10usize);
}