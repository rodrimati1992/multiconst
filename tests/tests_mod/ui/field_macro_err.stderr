error: expected either an untyped numeric literal or an identifier
 --> tests/tests_mod/ui/field_macro_err.rs:4:19
  |
4 |     let _: field!("hello");
  |                   ^^^^^^^
  |
  = note: this error originates in the macro `field` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected either an untyped numeric literal or an identifier
 --> tests/tests_mod/ui/field_macro_err.rs:5:19
  |
5 |     let _: field!(0x100);
  |                   ^^^^^
  |
  = note: this error originates in the macro `field` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected either an untyped numeric literal or an identifier
 --> tests/tests_mod/ui/field_macro_err.rs:6:19
  |
6 |     let _: field!(-1);
  |                   ^
  |
  = note: this error originates in the macro `field` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected either an untyped numeric literal or an identifier
 --> tests/tests_mod/ui/field_macro_err.rs:7:19
  |
7 |     let _: field!(10usize);
  |                   ^^^^^^^
  |
  = note: this error originates in the macro `field` (in Nightly builds, run with -Z macro-backtrace for more info)
